ashpd-demo/data/resources/ui/global_shortcuts.ui

```diff
@@ -0,0 +1,120 @@
+                    <property name="title" translatable="yes">Shortcuts</property>
+                  </object>
+                </child>
+              </object>
+            </child>
+            <child>
+              <object class="GtkBox">
+                <property name="orientation">horizontal</property>
+                <child>
+                  <object class="GtkButton">
+                    <property name="label" translatable="yes">_Start Session</property>
+                    <property name="use-underline">True</property>
+                    <property name="halign">start</property>
+                    <property name="action-name">global_shortcuts.start_session</property>
+                    <style>
+                      <class name="pill" />
+                      <class name="suggested-action" />
+                    </style>
+                  </object>
+                </child>
+                <child>
+                  <object class="GtkButton">
+                    <property name="label" translatable="yes">_Close Session</property>
+                    <property name="use-underline">True</property>
+                    <property name="halign">start</property>
+                    <property name="sensitive">False</property>
+                    <property name="action-name">global_shortcuts.stop</property>
+                    <property name="halign">end</property>
+                    <property name="hexpand">True</property>
+                    <style>
+                      <class name="pill" />
+                      <class name="destructive-action" />
+                    </style>
+                  </object>
+                </child>
+              </object>
+            </child>
+            <child>
+              <object class="AdwPreferencesGroup" id="activations_group">
+                <property name="title" translatable="yes">Activations</property>
+                <property name="visible">False</property>
+                <property name="margin-top">24</property>
+                <child>
+                  <object class="GtkLabel" id="activations_label">
+                    <property name="valign">center</property>
+                  </object>
+                </child>
+                <child>
+                  <object class="AdwActionRow">
+                    <property name="title" translatable="yes">Binding changes count</property>
+                    <child>
+                      <object class="GtkLabel" id="rebind_count_label">
+                        <property name="valign">center</property>
+                        <style>
+                          <class name="dim-label" />
+                        </style>
+                      </object>
+                    </child>
+                  </object>
+                </child>
+              </object>
+            </child>
+            <child>
+              <object class="AdwPreferencesGroup" id="response_group">
```

Comment:
The respnose group seems to be completely unused, I would suggest moving the binding_changes_count there and maybe adding a row that displays the shortcut that was triggered or something like that